src/main/webapp/WEB-INF/jsp/content/letter/list.jsp

```diff
@@ -43,7 +43,10 @@
                                 <a href="<spring:url value='/content/letter/edit/${letter.id}' />">"<c:out value='${letter.text}' />"</a>
                             </td>
                             <td>
-                                ${letter.diacritic}
+                                <c:choose>
+                                	<c:when test="${letter.diacritic}">Yes</c:when>
```

Comment:
I suggest using `<fmt:message key="yes" />` instead of hardcoding "Yes"/"No".

Reply:
The change is made. Thank you for the suggestion. :)